src/server/pfs/server/driver.go

```diff
@@ -333,9 +333,15 @@ func (d *driver) getAccessLevel(pachClient *client.APIClient, repo *pfs.Repo) (a
 	if err != nil {
 		return auth.Scope_NONE, err
 	}
-	if who.IsAdmin {
-		return auth.Scope_OWNER, nil
+
+	if who.AdminGrant != nil {
```

Comment:
This method lives in the PFS server but it seems like a re-implementation of Authorize in the auth server - especially if we wire up group membership it probably makes sense to consolidate this logic in the auth server.